pkg/validator/am0001/default_channel.go

```diff
@@ -84,3 +90,38 @@ func (d *DefaultChannel) isListedInChannels(channels *[]v1alpha1.Channel, defaul
 	msg := fmt.Sprintf("The defaultChannel '%v' is not part of the listed channelNames: %v.", defaultChannel, channelNames)
 	return d.Fail(msg)
 }
+
+func (d *DefaultChannel) isListedInBundleChannel(defaultChannel string, bundles []*registry.Bundle) validator.Result {
+	var message []string
+	for _, bundle := range bundles {
+		if bundle.Annotations.DefaultChannelName != defaultChannel && bundle.Annotations.DefaultChannelName != "" {
+			msg := fmt.Sprintf("The defaultChannel is '%v' is not matching operators.operatorframework.io.bundle.channel.default.v1 '%v'.",
+				defaultChannel, bundle.Annotations.DefaultChannelName,
+			)
+			message = append(message, msg)
+		}
+
+		channels := strings.Split(bundle.Annotations.Channels, ",")
```

Comment:
Can we be 100% sure that there are no spaces in the annotation? Would it be safer to do `strings.Split(strings.ReplaceAll(bundle.Annotations.Channels," ","") , ",")`

Reply:
The registry implements the channels in the same way I have Implemented.
Ref: https://github.com/operator-framework/operator-registry/blob/master/pkg/registry/bundle.go#L68

Reply:
Kubernetes annotations are guaranteed to not contain whitespace (https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set).